draft-gpew-priv-ppm.md

```diff
@@ -559,7 +559,7 @@ struct {
 
 * `aggregator_config_id` is equal to `HpkeConfig.id`, where `HpkeConfig` is the
   key config of the aggregator receiving the input share.
-* `enc` is the encapsulated HPKE context, used by the aggregator to decrypt its
+* `enc` is the HPKE encapsulated key, used by the aggregator to decrypt its
```

Comment:
This is sometimes referred to as the "encapsulated context", since it's used to construct an HPKE context. That said, I'm fine with this change.